tests/test_slogpy_console.py

```diff
@@ -0,0 +1,40 @@
+"""The purpose of the code is to give debug method a unit test to make sure that they are working properly"""
+from slogpy.slog import Slog as slog  # noqa: N813
+
+
+def setup_function():
+    """set the Default to no log file, since only interested in console output"""
```

Comment:
nit: surprised the linter didn't complain, but it is customary to start with a capital letter.